src/distribution/cauchy.rs

```diff
@@ -99,6 +99,22 @@ impl ContinuousCDF<f64, f64> for Cauchy {
     fn cdf(&self, x: f64) -> f64 {
         (1.0 / f64::consts::PI) * ((x - self.location) / self.scale).atan() + 0.5
     }
+
+    /// Calculates the survival function for the
+    /// cauchy distribution at `x`
+    ///
+    /// # Formula
+    ///
+    /// ```ignore
+    /// (1 / π) * arctan(-(x - x_0) / γ) + 0.5
+    /// ```
+    ///
+    /// where `x_0` is the location and `γ` is the scale.
+    /// note that this is identical to the cdf except for
+    /// the negative argument to the arctan function
+    fn sf(&self, x: f64) -> f64 {
+        (1.0 / f64::consts::PI) * (-((x - self.location) / self.scale)).atan() + 0.5
```

Comment:
Would it be cleaner to write `* (self.location - x) / self.scale).atan()` to reduce the noise?

Reply:
yeah great catch! I made this change and pushed it to branch - definitely cleans it up